apps/web/package.json

```diff
@@ -49,8 +49,9 @@
     "graphql": "^16.8.1",
     "idb-keyval": "^6.2.1",
     "lexical": "^0.12.4",
-    "next": "^14.0.3",
+    "next": "^14.0.1",
     "next-themes": "^0.2.1",
+    "pino-pretty": "^10.2.3",
```

Comment:
have we used this anywhere?

Reply:
@bigint 
For some reason it would not compile without this package in app dir mode.

Reply:
Is there a docs mentioned anywhere?